aider.el

```diff
@@ -63,9 +68,11 @@ This function can be customized or redefined by the user."
     ("z" "Switch to Aider Buffer" aider-switch-to-buffer)
     ("l" "Clear Aider" aider-clear)
     ("s" "Reset Aider" aider-reset)
+    ("x" "Exit Aider" aider-exit)
```

Comment:
Thanks for adding this. Make sense to have a exit command. Could be better than just kill the buffer which I was doing. Especially there is some finalize work aider did during /exit